PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/AzureActionsComponent.java

```diff
@@ -78,12 +82,16 @@
     public AzureActionsComponent() {
         DefaultLoader.setPluginComponent(this);
         DefaultLoader.setUiHelper(new UIHelperImpl());
-        DefaultLoader.setIdeHelper(new IDEHelperImpl());
-        Node.setNode2Actions(NodeActionsMap.node2Actions);
+        DefaultLoader.setIdeHelper(createIDEHelper());
         SchedulerProviderFactory.getInstance().init(new AppSchedulerProvider());
         MvpUIHelperFactory.getInstance().init(new MvpUIHelperImpl());
 
-        HDInsightLoader.setHHDInsightHelper(new HDInsightHelperImpl());
+        Map<Class<? extends Node>, ImmutableList<Class<? extends NodeActionListener>>> node2Actions = new HashMap<>();
+        for (NodeActionsMap nodeActionsMap : Extensions.getExtensions(NodeActionsMap.EXTENSION_POINT_NAME)) {
```

Comment:
Try `ServiceManager` for provider getting

Reply:
Maybe I did not understand `ServiceManager` API, but here we collect several `NodeActionsMap` implementations, not a single instance.